README.rst

```diff
@@ -83,3 +83,42 @@ https://greentreesnakes.readthedocs.org/en/latest/
 Documentation of the various types of AST nodes that Bandit currently covers
 or could be extended to cover:
 https://greentreesnakes.readthedocs.org/en/latest/nodes.html
+
+Container Images
+----------------
+
+Bandit is available as a container image, built within the bandit repository
+using GitHub Actions. The image is available on gchr.io:
+
+```bash
+docker pull gcr.io/pycqa/bandit/bandit:py312-arm64
+```
+
+The image is built for the following architectures:
+
+* amd64
+* arm64
+
+The image is tagged with the Python version and architecture, for example:
+
+* py312-amd64
+* py312-arm64
+
+Python versions supported are:
+
+* 3.8 (py38-amd64)
+* 3.9 (py39-amd64)
+* 3.10 (py310-amd64)
+* 3.11 (py311-amd64)
+* 3.12 (py312-amd64)
```

Comment:
tl;dr:
* We should list supported versions without tag names or with both tag names
* I would prefer we provide *one* version of Python (until we have an overwhelming demand for more versions) and it only be latest 3.12
* I think we need a way to manually rebuild an image with the latest version of the base image to ensure security vulnerabilities are updated/patched in the image. So we'd need something like `<bandit-version>-<image-version-starting-at-0-or-1>-py<ver>-<arch>` for best effort of providing a secure image to users

---


Above we say `arm64` and `amd64` but here we're only listing the x86 versions.

Further, _why_ do we want to do this for all the different versions of Python? That seems... unnecessary. I know the AST, standard-library, etc. are all different across things but the AST isn't so radically backwards incompatible that only providing an image against 3.12 would be bad.

Finally, my last concern here is that by having this many images, imagine the scenario where someone discloses a CPython vulnerability that affects everything before 3.12.(latest). If we genuinely care about what we're providing, we should be able to republish against the latest versions of the alpine images with patched CPython, but I don't think we have a good way of doing that today with this setup.

Reply:
I am good with just providing 3.12

How I remove everything save 3.12 as amd64 / arm64 arch builds?

Reply:
I agree, just py3.12 is a nice choice. As for architectures, we never had to be concerned with it before, but now would need to with the base image. I guess now we might be excluding other architectures (https://wiki.alpinelinux.org/wiki/Architecture). That's probably okay, but who knows, there might be users on PPC64.

Reply:
There are very few people on ppc64 for the usecase we're carrying about here (CI/CD that relies primarily on container images). The ppc64 people are also usually very hostile (and have harassed people out of the python community) so I'm not inclined to support them anyway.